Library/Homebrew/utils/sorbet.rb

```diff
@@ -0,0 +1,17 @@
+# typed: strict
+# frozen_string_literal: true
+
+if ENV["HOMEBREW_SORBET_RUNTIME"]
+  require "utils/gems"
+  Homebrew.install_bundler_gems!
+  require "sorbet-runtime"
+else
+  begin
+    gem "sorbet-runtime"
```

Comment:
Not sure I understand what's going on here, can you explain?

Reply:
I don't think this is actually needed. Removed.